### DISCUSSION TITLE
An Idea: Use random numbers in LUA scripts.

### DISCUSSION THREAD

**Discussion by Flyer133** [CrashOverride85/zc95#33] [Ideas]
Hello CrashOverride85,
An idea for future devlopment: a lua function who return an integer between two values.
The scripts will be more unprevisible with this....
Regards

**Comment by CrashOverride85** [comment]
Hi, this is already possible using the math.random function in the Lua math library, e.g. to output a random number between 1 and 10:
```
    randomNumber = math.random(1, 10)
    print(randomNumber)
```
